applications/body_pose_estimation/README.md

```diff
@@ -17,6 +17,8 @@ The model is downloaded when building the application.
 
 This application downloads a pre-recorded video from [Pexels](https://www.pexels.com/video/a-woman-showing-her-ballet-skill-in-turning-one-footed-5385885/) when the application is built.
 
+> **_NOTE:_** For each dataset an user elects to use, the user is responsible for checking if the dataset license is fit for the intended purpose.
```

Comment:
A couple of questions:
- Is the Pexels "twirl" MP4 the only dataset provided with this app, or are there others?
- Under what license is the Pexels video distributed?
- Can we provide a link to the Pexels video in the README and/or in `metadata.json`?

Reply:
@tbirdso 

- Yes, the "twirl" mp4 is the only dataset we are making use of in this application
- good point, I've added a link to the pexels license terms 
- sure, there's already a link to the video in the README.  Just updated the metadata.json to reference the video as well

Reply:
Thanks! Current language suggests multiple datasets, suggest a small update:

```suggestion
> **_NOTE:_** The user is responsible for checking if the dataset license is fit for the intended purpose.
```